ddtrace/contrib/django/utils.py

```diff
@@ -128,9 +127,9 @@ def get_request_uri(request):
     # combining the url parts. We returns a byte string when all url parts are
     # byte strings.
     # https://github.com/python/cpython/blob/02d126aa09d96d03dcf9c5b51c858ce5ef386601/Lib/urllib/parse.py#L111-L125
-    if PY3 and not all(isinstance(value, binary_type) or value is None for value in urlparts.values()):
```

Comment:
cc @majorgreys this was added for a specific reason, no?

Reply:
I suppose maybe it doesn't matter anymore if we stop using `urlunparse`?

If so, we can likely remove the comment starting on L123